src/ast/codegen_helper.h

```diff
@@ -10,22 +10,27 @@ inline bool needMemcpy(const SizedType &stype)
   return stype.IsAggregate() || stype.IsTimestampTy() || stype.IsCgroupPathTy();
 }
 
-inline bool shouldBeOnStackAlready(const SizedType &type)
+// BPF memory is memory that the program can access with a regular
+// dereference. This could mean the value is on the stack, a map, or
+// maybe something else (like BPF arenas) in the future.
+//
+// The means that a bpf_probe_read_*() is _NOT_ required.
```

Comment:
Sorry, one more minor typo

```suggestion
// This means that a bpf_probe_read_*() is _NOT_ required.
```

Reply:
Ha, working too fast...